deploy/central/images/bc-ods-build-python.yml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: Template
```

Comment:
Note that the new structure does not use templates anymore, but plain `BuildConfig` resources.